component/autoscaler.jsonnet

```
@@ -153,6 +228,16 @@ if params.autoscaling.enabled then
       'machine_autoscalers']: machineAutoscalers,
     [if priorityExpanderConfigmap != null then
       'priority_expander_configmap']: priorityExpanderConfigmap,
+    [if params.autoscaling.schedule.enabled then
+      'downscaler_service_account']: autoscalerServiceAccount,
+    [if params.autoscaling.schedule.enabled then
+      'downscaler_cluster_role']: autoscalerClusterRole,
+    [if params.autoscaling.schedule.enabled then
+      'downscaler_cluster_role_binding']: autoscalerClusterRoleBinding,
+    [if params.autoscaling.schedule.enabled then
+      'enable_downscaler_cronjob']: enableDownscalerCronJob,
+    [if params.autoscaling.schedule.enabled then
+      'disable_downscaler_cronjob']: disableDownscalerCronJob,
```

You could make this less verbose by either adding all objects in a single output file:

```suggestion
    [if params.autoscaling.schedule.enabled then 
      'downscale_cronjobs']: [
        autoscalerServiceAccount,
        autoscalerClusterRole,
        autoscalerClusterRoleBinding,
        enableDownscalerCronJob,
        disableDownscalerCronJob,
      ],
```

Or by defining a separate object which you append to the default config when `params.autoscaling.schedule.enabled` is true:

```
if params.autoscaling.enabled then
  {
    // existing outputs
  } + if params.auotscaling.schedule.enabled then {
    downscaler_service_account: autoscalerServiceAccount,
    downscaler_cluster_role: autoscalerClusterRole,
    downscaler_cluster_role_binding: autoscalerClusterRoleBinding,
    enable_downscaler_cronjob: enableDownscalerCronJob,
    disable_downscaler_cronjob: disableDownscalerCronJob,
  }
```